### PR TITLE
Navitia-Geovelo mapping update

### PR DESCRIPTION
**Pull request by wassimbenaissa**
Following several customer feedback, I have adjusted the Navitia geolveo alternatives mapping.

### CODE REVIEW

**Review by kadhikari — APPROVED** [review]

File: source/jormungandr/jormungandr/street_network/geovelo.py [hunk 1]
```diff
@@ -188,11 +188,11 @@ def _call_geovelo(self, url, method=requests.post, data=None):
     @staticmethod
     def get_geovelo_tag(geovelo_response):
         if geovelo_response['title'] == 'RECOMMENDED':
-            return 'rapid'
+            return 'balanced'
         elif geovelo_response['title'] == 'FASTER':
```
Comment (stifoon) [inline source/jormungandr/jormungandr/street_network/geovelo.py:192]: I would have mapped "rapid" with this one : rapid is a well-known tag in Navitia, not shorter ;)

File: source/jormungandr/jormungandr/street_network/geovelo.py [hunk 2]
```diff
@@ -188,11 +188,11 @@ def _call_geovelo(self, url, method=requests.post, data=None):
...
             return 'shortest'
         elif geovelo_response['title'] == 'SAFER':
-            return 'balanced'
+            return 'comfort'
         elif geovelo_response['title'] == 'BIS':
             return 'comfort'
```
Comment (stifoon) [inline source/jormungandr/jormungandr/street_network/geovelo.py:197]: so there could be 2 comfort solutions ?

Comment (wassimbenaissa) [reply source/jormungandr/jormungandr/street_network/geovelo.py:197]: Yes :)

File: source/jormungandr/jormungandr/street_network/geovelo.py (hunk 1 above)
Comment (wassimbenaissa) [reply source/jormungandr/jormungandr/street_network/geovelo.py:192]: Yep you'r right, but our front team use the "shortest" tag to highlight the "itinéraire rapide"

**Review by stifoon — APPROVED** [review]

**Review by pbench — APPROVED** [review]

**Review by azime — APPROVED** [review]

**Pull request merged**